app/views/finders/_show_header.html.erb

```diff
@@ -7,7 +7,6 @@
           heading_level: 1,
           font_size: "xl",
           margin_bottom: 4,
-          inverse: inverse,
```

Comment:
Sorry, just noticed this - what was this change for?

Reply:
I couldn't replicate a view where there is an inverse header, so I thought it was an error.

Reply:
I think I'd be more comfortable with this change supported by some more investigation. Since this is unrelated I suggest not deleting this line as part of this PR.

Reply:
OK I've reverted that and also the other two titles further down the page where I'd also removed the inverse option.

Reply:
Great, thanks. I've spotted an unrelated problem with this app but I'd like to get your changes out first, I know you've just finished for today so apologies am going to merge this PR (if I can get the tests to pass).